Usecase

JoseAlba opened the pull request.

JoseAlba committed:
Request delievery of materials to location

Usecase 1

JoseAlba committed:
testing

PatrickMalolepszy reviewed (changes requested):
There are quite a few typos in the use case, open in an editor with spell check for help fixing.

Also are we supposed to have all of the headings (scope, level, stakeholders & interests, etc...) in a brief use case? If so I need to change mine!

JoseAlba committed:
Fixed Typos

There were quite a few typos in the use case I opened it in editor and fixed the typos.

PatrickMalolepszy reviewed (approved):
Good stuff - thanks Jose.

PatrickMalolepszy committed:
Merge branch 'master' into usecase
